src/sbuf/Spaces.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#ifndef SQUID_SRC_SBUF_SPACES_H
+#define SQUID_SRC_SBUF_SPACES_H
+
+#include "sbuf/SBuf.h"
+
+const SBuf spaces(size_t count);
```

Comment:
FWIW, I will try to find the time to check whether this can become a constexpr (which may even improve some code in callers). If I succeed, I will fix the return type (extra/unwanted `const`), the lack of documentation for this new global function, and NoNewGlobals implementation violation at that time.

Reply:
Based on our conversation, I'd hold off on this experiment; spaces (or its successor) will likely need to hold some state